Header header 
uint32 sq
time stamp
string frame_id
uint32 image_width
uint32 image_height
uint32 blob_count
Blob[] blobs
